metadata-transformer/src/test/java/no/unit/nva/metadata/service/testdata/ValidDoiArgumentsProvider.java

```diff
@@ -1,32 +1,64 @@
 package no.unit.nva.metadata.service.testdata;
 
+import no.unit.nva.metadata.Citation;
+import no.unit.nva.metadata.DcTerms;
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.ArgumentsProvider;
 
 import java.net.URI;
-import java.util.List;
-import java.util.Map;
 import java.util.stream.Stream;
 
 public class ValidDoiArgumentsProvider implements ArgumentsProvider {
 
+    public static final String DC_IDENTIFIER = DcTerms.IDENTIFIER.getDcLocalName();
+    public static final String CITATION_DOI = Citation.DOI.getProperty();
```

Comment:
Related to my previous comment: here getProperty is used. It's not immediately clear to me why getDcLocalName() and getProperty() are named differently, nor from only the method name, what it gets. That might be because I'm not used to the conventions you have applied here, just mentioning my immediate impression.

Reply:
Your intuition here was entirely correct, what we are extracting is the "Meta Tag name", either "citation_doi", "DC.identifier" and as such I changed the name to exactly this.